01.create_secFiles.yaml

```diff
@@ -21,7 +21,7 @@
     jwtTokenOnly: false
     brokerOnly: false
     # srv_host_list: "{{ groups['broker']['private_ip']|join(',') }}"
-    srv_host_list: "{{ groups['broker'] | map('extract', hostvars, ['private_ip']) | join(',') }}"
+    srv_host_list: "{{ groups['broker'] | map('extract', hostvars, ['private_ip']) | join(',') if 'broker' in groups else None }}"
```

Comment:
Let's reuse "srv_component" vraiable
```
if srv_component in groups else None
```

Reply:
Corrected in https://github.com/datastax/pulsar-ansible/pull/14/commits/fb6a26fb024b451b7fb568fc9ef59fd9c1278dcc